crates/forge-runner/src/gas/report.rs

```diff
@@ -119,11 +121,26 @@ impl Display for ReportData {
     }
 }
 
-fn get_contract_name(contracts_data: &ContractsDataStore, class_hash: ClassHash) -> ContractName {
-    contracts_data
-        .get_contract_name(&class_hash)
-        .map_or("forked contract", |name| name.0.as_str())
-        .to_string()
+impl Display for ContractId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let name = match self {
+            ContractId::TestContract(name) => format!("{name} Contract"),
+            ContractId::ForkedContract(class_hash) => {
+                format!(
+                    "forked contract\n(class hash: {})",
```

Comment:
In many cases people will use addresses in order to interact with contract from forks, so imo it would be handy here too. Displaying only class hash adds an extra step because people have to look it up. Wdyt?